modules/utxo-lib/src/payments/p2tr.ts

```diff
@@ -18,6 +20,19 @@ const BITCOIN_NETWORK = networks.bitcoin;
 const H = Buffer.from('50929b74c1a04954b78b4b6035e97a5e078a5a0f28ec96d547bfee9ace803ac0', 'hex');
 const EMPTY_BUFFER = Buffer.alloc(0);
 
+function isOrdinaryPubkey(pubKey: Uint8Array): boolean {
+  if (pubKey.length !== 33) return false;
+  try {
+    return !!necc.Point.fromHex(pubKey);
```

Comment:
haven't we decoded from hex here already? shouldn't this be `fromBuffer` instead?

Reply:
I was basically reusing the check we use for keys of length 32. Also uses `!!necc.Point.fromHex(pubKey)` and looks like the fromHex method takes this type `declare type Hex = Uint8Array | string;` so either a string or Uint8Array (what we're passing)


https://github.com/BitGo/BitGoJS/blob/862850781b2e8b36c71608c5ae71424b9ebe9dee/modules/utxo-lib/src/noble_ecc.ts#L29-L36